redfish-core/lib/memory.hpp

```diff
@@ -452,6 +454,12 @@ inline void
         asyncResp->res.jsonValue[jsonPtr]["DataWidthBits"] = *memoryDataWidth;
     }
 
+    if (correctableError != nullptr)
```

Comment:
This is property is not aprt of Memory schema .
Please refer https://github.com/AMDESE/bmcweb/blob/integ_sp7/redfish-core/schema/dmtf/json-schema/MemoryMetrics.v1_7_3.json#L270